Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -520,7 +520,6 @@ namespace Babylon
                     auto callback{std::move(m_requestAnimationFrameCallback)};
                     callback({});
                 }
-                GetFrameBufferManager().Reset();
```

Comment:
Without a reset or a recycling mechanism, the view IDs will increase without bound over time, eventually exceeding `bgfx::getCaps()->limits.maxViews`, right? Or is there another mechanism preventing this from happening?